src/core/server/saved_objects/migrations/core/document_migrator.ts

```diff
@@ -67,7 +67,7 @@ import { SavedObjectMigrationFn, SavedObjectMigrationMap } from '../types';
 import { DEFAULT_NAMESPACE_STRING } from '../../service/lib/utils';
 import { LegacyUrlAlias, LEGACY_URL_ALIAS_TYPE } from '../../object_types';
 
-const DEFAULT_MINIMUM_CONVERT_VERSION = '8.0.0';
+const DEFAULT_MINIMUM_CONVERT_VERSION = '7.13.0';
```

Comment:
See https://github.com/elastic/kibana/pull/97222#discussion_r619870721, just commenting here too so it doesn't accidentally get overlooked-- I think we need to change this back
```suggestion
const DEFAULT_MINIMUM_CONVERT_VERSION = '8.0.0';
```